src/main/java/uk/gov/hmcts/reform/em/hrs/controller/HearingRecordingController.java

```diff
@@ -145,24 +149,17 @@ public ResponseEntity getSegmentBinary(@PathVariable("recordingId") UUID recordi
                                            @PathVariable("segment") Integer segmentNo,
                                            HttpServletRequest request,
                                            HttpServletResponse response) {
-
-        LOGGER.info("received request to download recording for case ({}) segment ({})", recordingId, segmentNo);
-
-        Map<String, String> segmentDetails = downloadService.getDownloadInfo(recordingId, segmentNo);
-        String filename = segmentDetails.get("filename");
-
-        response.setHeader(HttpHeaders.CONTENT_TYPE, segmentDetails.get("contentType"));
-        response.setHeader(
-            HttpHeaders.CONTENT_LENGTH,
-            segmentDetails.get("contentLength")
-        );//will be overriden if partial request
-
```

Comment:
so is this not required anymore? download was failing when we weren't setting these headers